eth_block.go

```diff
@@ -0,0 +1,362 @@
+package ipldeth
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	cid "github.com/ipfs/go-cid"
+	node "github.com/ipfs/go-ipld-format"
+
+	"github.com/ethereum/go-ethereum/common"
+	types "github.com/ethereum/go-ethereum/core/types"
+	rlp "github.com/ethereum/go-ethereum/rlp"
+)
+
+// EthBlock (eth-block, codec 0x90), represents an ethereum block header
+type EthBlock struct {
+	*types.Header
```

Comment:
Let's leave it for a future version overhauling. I am keen on having all the ETH - IPLD types represented in this plugin as soon as I can.